tools/style/uncrustify_run_dir.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
```

Comment:
Good old plain `!/bin/sh` is POSIX, no trickery with /usr/bin/env is needed to find a bourne shell